Content.Server/Nutrition/EntitySystems/FoodSequenceSystem.cs

```diff
@@ -114,8 +120,12 @@ public override void Initialize()
 
     private void OnInteractUsing(Entity<FoodSequenceStartPointComponent> ent, ref InteractUsingEvent args)
     {
+        if (HasComp<EntityStorageComponent>(args.Used)
+            || HasComp<StorageComponent>(args.Used))
+            return; // Prevent Backpacks/Pet Carriers
```

Comment:
```suggestion
        if (HasComp<EntityStorageComponent>(args.Used)
            || HasComp<StorageComponent>(args.Used)
            && !ent.Comp.AcceptAll)
            return; // Goobstation - Prevent Backpacks/Pet Carriers
```
Just to make sure accept all actually... accepts all.

Reply:
This defeats the purpose. My code works as is. There are no foods that have Storage nor EntityStorage components on them. This is a blanket blacklist of those items in general. I am not applying your code.